great_expectations/render/renderer/site_builder.py

```diff
@@ -430,33 +430,54 @@ def get_calls_to_action(self):
                 drivername,
             )
 
-        buttons = [
-            (
+        return {
+            "header": "To continue exploring Great Expectations...",
+            "buttons": self._get_call_to_action_buttons(telemetry)
+        }
+
+    def _get_call_to_action_buttons(self, telemetry):
+        """
+        Build project and user specific calls to action buttons.
+
+        This can become progressively smarter about project and user specific
+        calls to action.
+        """
+        create_expectations = CallToActionButton(
                 "Create Expectations",
                 "https://docs.greatexpectations.io/en/latest/getting_started/create_expectations.html"
-            ),
-            (
-                "Play with Validations",
-                "https://docs.greatexpectations.io/en/latest/getting_started/pipeline_integration.html"
-            ),
-            (
-                "Customize Data Docs",
-                "https://docs.greatexpectations.io/en/latest/reference/data_docs_reference.html#customizing-data-docs"
-            ),
-            # TODO gallery does not yet exist
-            # (
-            #     "Great Expectations Gallery",
-            #     "https://greatexpectations.io/gallery"
-            # )
-        ]
+            )
+        validation_playground = CallToActionButton(
+            "Play with Validations",
+            "https://docs.greatexpectations.io/en/latest/getting_started/pipeline_integration.html"
+        )
+        customize_data_docs = CallToActionButton(
+            "Customize Data Docs",
+            "https://docs.greatexpectations.io/en/latest/reference/data_docs_reference.html#customizing-data-docs"
+        )
+        # TODO gallery does not yet exist
+        # gallery = CallToActionButton(
+        #     "Great Expectations Gallery",
+        #     "https://greatexpectations.io/gallery"
+        # )
+
+        results = []
+
+        expectations_store = self.data_context.stores["expectations_store"]
+        if not expectations_store.list_keys():
+            # TODO this needs testing as complexity increases probably using mocked DataContext
```

Comment:
This is a bit tricky, since the profiling will make it look like you have expectations - maybe filter out ones called `BasicDatasetProfiler`?